### PR TITLE
fix issue 740.

### PR DESCRIPTION
**Pull request by redbopo**
fix [740](https://github.com/lcompilers/lpython/issues/740)

### CODE REVIEW

File: tests/reference/asr-test_unsupported_type-0d813dd.stderr [hunk 1]
```diff
@@ -0,0 +1,5 @@
+semantic error: Unsupported type annotation: i128
+ --> tests/errors/test_unsupported_type.py:2:5
+  |
+2 |     i: i128
+  |     ^^^^^^^
```
Comment (namannimmo10) [inline tests/reference/asr-test_unsupported_type-0d813dd.stderr:5]: Can you try to improve the message like this?
```suggestion
2 |     i: i128
  |        ^^^^ 
```

Comment (certik) [reply tests/reference/asr-test_unsupported_type-0d813dd.stderr:5]: I think that can be done in a subsequent PR. This PR fixes the main problem to raise an error, which now works.

**Review by certik — APPROVED** [review]
I think this is fine. It changes the `symbol_get_past_external` function to pass through a null pointer. Alternatively we would add an assert here and fix it in the caller. I think this is fine for now.

**Comment by redbopo** [comment]
Thanks @namannimmo10  and @certik , I will try a  better error message later.

**Pull request merged**